script/ci/cache_prepare.sh

```diff
@@ -38,9 +38,9 @@ run() {
   eval $2;
 }
 
-run "bundle exec rake db:migrate webdrivers:chromedriver:update webdrivers:geckodriver:update"
+run "bundle exec rake db:create db:migrate webdrivers:chromedriver:update webdrivers:geckodriver:update"
 
-run "for i in {1..3}; do (cd frontend; npm install && break || sleep 15;) done"
+run "cd frontend; npm install ; cd -"
```

Comment:
we could make `run` eval in a separate context (e.g. `(eval $2)`) that way we don't need to cd back after commands

Reply:
Sure, and we don't need that `run` prefix either. Not sure what it was for.